doc/src/manual/documentation.md

```diff
@@ -621,13 +621,14 @@ the Julia documentation itself. For example:
 
 ```julia
 """
-    accumulate!(op, y, x)
+    tryparse(type, str; base)
 
-Cumulative operation `op` on a vector `x`, storing the result in `y`. See also [`accumulate`](@ref).
+Like [`parse`](@ref), but returns either a value of the requested type,
+or [`nothing`](@ref) if the string does not contain a valid number.
 """
 ```
 
-This will create a link in the generated docs to the `accumulate` documentation
+This will create a link in the generated docs to the `parse` documentation
```

Comment:
"and `nothing`"? Also should the `parse` have a `@ref` in this sentence (it kinda shows what is described with an example then)

Reply:
I hesitated for `nothing` as it doesn't fall in the one the two categories below (mutating/non-mutating and similar-seeming functions), but no problem to do it. Good idea to add a ref link here too.